arch/wasm/bits/posix.h

```diff
@@ -0,0 +1,2 @@
+#define _POSIX_V6_LP64_OFF64  1
+#define _POSIX_V7_LP64_OFF64  1
```

Comment:
wasm32 is ILP32, so it looks like these should be ILP32_OFFBIG rather than LP64_OFF64, for wasm32.

Reply:
Hm, I just copied that blindly from the x32 port - which is also ILP32, surely? So it looks like a bug in `arch/x32/bits/posix.h` that I didn't notice or think about. `off_t` is complicated, I was just hoping I wouldn't have to check out the specs for it!

I'll raise it tomorrow on the Musl mailing list for the x32 port.